src/EnergyPlus/Humidifiers.cc

```diff
@@ -728,7 +728,7 @@ namespace Humidifiers {
 			}
 
 			if ( !HardSizeNoDesRun ) {
-				NomCapVolDes = MassFlowDes * ( OutletHumRatDes - InletHumRatDes ) / AirDensity;
+				NomCapVolDes = MassFlowDes * ( OutletHumRatDes - InletHumRatDes ) / RhoH2O( InitConvTemp );

```

Comment:
@Nigusse Could you add to the unit tests in Humidifiers.unit.cc to verify that this is getting computed correctly?